app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -2096,6 +2098,7 @@ public void run() {
      */
     private void openLogin() {
         Intent i = new Intent(this, LoginActivity.class);
+        i.putExtra("serverUrl", Constants.serverUrl);
```

Comment:
I think it will be better to add all key's in separate class (may be in `Constant.java`)